libpod/runtime_img.go

```diff
@@ -313,15 +313,22 @@ func (r *Runtime) LoadImageFromSingleImageArchive(ctx context.Context, writer io
 		func() (types.ImageReference, error) {
 			return layout.NewReference(inputFile, "")
 		},
+		func() (types.ImageReference, error) {
+			// This item needs to be last to break out of loop and report meaningful error message
+			return nil, errors.New("loader failed to determine payload format, could be corrupted")
```

Comment:
How about: `the specified input does not match any of the supported image formats (oci-archive, oci-dir, docker-archive, docker-dir)`?

Reply:
Can you extend the `"podman load invalid file"` test in `test/systemd/120-load.bats` to string match the error message?